client/login/wp-login/components/heading-logo.tsx

```diff
@@ -1,14 +1,22 @@
+import akismetLogo from 'calypso/assets/images/icons/akismet-logo.svg';
 import studioAppLogo from 'calypso/assets/images/icons/studio-app-logo.svg';
 import { isStudioAppOAuth2Client } from 'calypso/lib/oauth2-clients';
 import { useSelector } from 'calypso/state';
 import { getCurrentOAuth2Client } from 'calypso/state/oauth2-clients/ui/selectors';
 
-const HeadingLogo = () => {
+interface Props {
+	isFromAkismet?: boolean;
+}
+
+const HeadingLogo = ( { isFromAkismet }: Props ) => {
 	const oauth2Client = useSelector( getCurrentOAuth2Client );
 
-	const logo = isStudioAppOAuth2Client( oauth2Client ) ? (
-		<img className="wp-login__heading-logo-studio" src={ studioAppLogo } alt="Studio App Logo" />
-	) : null;
+	let logo = null;
+	if ( isStudioAppOAuth2Client( oauth2Client ) ) {
+		logo = <img src={ studioAppLogo } alt="Studio App Logo" />;
+	} else if ( isFromAkismet ) {
+		logo = <img src={ akismetLogo } alt="Akismet Logo" />;
+	}
```

Comment:
Probably not worth the conditioning here, not this way, as it's only for setting the alt text. Can clean up in the end, not sure what final form this will need.